File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-insert-text-into-word-documents_11.cs
            else 
                if (currentSelection.Type == Word.WdSelectionType.wdSelectionNormal)
                { 
                    // Move to start of selection.
                    if (Application.Options.ReplaceSelection)
                    { 
                        object direction = Word.WdCollapseDirection.wdCollapseStart;
                        currentSelection.Collapse(ref direction);
                    }
                    currentSelection.TypeText("Inserting before a text block. ");
                    currentSelection.TypeParagraph();
                }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-insert-text-into-word-documents_11.vb
            ElseIf .Type = Word.WdSelectionType.wdSelectionNormal Then
                ' Move to start of selection.
                If Application.Options.ReplaceSelection Then
                    .Collapse(Direction:=Word.WdCollapseDirection.wdCollapseStart)
                End If
                .TypeText("Inserting before a text block. ")
                .TypeParagraph()